Source/CheatSheet/cheatsheet.md

```diff
@@ -62,79 +63,106 @@ The retrieve expression is the central construct for accessing clinical informat
 
 2) [terminology filter part](https://cql.hl7.org/02-authorsguide.html#filtering-with-terminology) : the retrieve expression allows the results to be filtered using terminology, including value sets, code systems, or by specifying a single code.
 
+- The example below filters the results to only Conditions whose code is in the "Acute Pharyngitis" value set.
+
 ```cql
-[Condition: "Acute Pharyngitis"] where Acute Pharyngitis is the value set.
+[Condition: "Acute Pharyngitis"]
 ```
 
-TODO: This isn't right, it's not an example of a function, the link isn't to the function declaration section of the spec, and functions don't have to be quoted identifiers :)
-3) [function](https://cql.hl7.org/02-authorsguide.html#statements) : A function in CQL is a named expression that is allowed to take any number of arguments. A function can be invoked directly by name. A colon must end the quoted identifier.
+- The example below returns Conditions with class in the "Inpatient Encounters" value set.
 
-TODO: Do we want to introduce the code path here? If we do, it needs to be done correctly, (i.e. it shouldn't be using an `=`, it should be using either `in` or `~`, and if it uses `~`, it should be a direct-reference code, not a value set)
 ```cql
-define "Inpatient Encounters": 
-[Encounter: class = "Inpatient Encounter"]
+valueset "Inpatient Encounters": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.666.5.307'
+[Condition: class in "Inpatient Encounters"]
 ```
 
-TODO: This should probably just be Queries, not "Alias functionality"
-### [**Alias functionality**](https://cql.hl7.org/02-authorsguide.html#queries)
+- The example below retrieves Conditions with codes equivalent to the "Diabetes Code" direct-reference code.
 
-A query construct often begins by introducing an alias for the primary source.
- 
 ```cql
-["Encounter": "Inpatient"] E
-  where E.period during "Measurement Period"
+code "Diabetes Code": '123' from "LOINC"
+[Condition: code ~ "Diabetes Code"]
 ```
 
-### [**Full Query Syntax**](https://cql.hl7.org/02-authorsguide.html#full-query)
 
-The clauses described in the clauses section later must appear in the correct order in order to specify a valid CQL query. The general order of clauses is:
+### [**Function Syntax**](https://cql.hl7.org/19-l-cqlsyntaxdiagrams.html#function)
 
-```bnf
-<primary-source> <alias>
-  <with-or-without-clauses>
-  <where-clause>
-  <return-clause>
-  <sort-clause>
+A function in CQL is a named expression that is allowed to take any number of arguments. A function can be invoked directly by name. A colon must end the quoted identifier.
```

Comment:
The function name needs to be an identifier (either simple or quoted), so I would suggest removing the bit about the colon entirely, since that's true of any define expression and isn't unique to functions.

```suggestion
A function in CQL is a named expression that is allowed to take any number of arguments. A function can be invoked directly by name.
```